pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.black]
+exclude = '''
```

Comment:
Add exclusions for conflicts with pep8

Reply:
should be good, black is pep8 compliant. It disagrees with flake8 on a whitespace thing, but exception for that added in `tox.ini`.